components/automate-cli/cmd/chef-automate/automateConstants.go

```diff
@@ -72,4 +72,17 @@ const (
 	`
 
 	SUDO_PASSWORD_CMD = `echo "%s" | sudo -S bash -c "`
+
+	STOP_FE_SERVICES_CMD = `sudo chef-automate stop`
```

Comment:
use systemctl to stop the services instead of `chef-automate stop` as this throws an error if its already stopped or in error state

Reply:
done